libs/net/entities/Minister.cs

```diff
@@ -13,6 +13,10 @@ public class Minister : BaseType<int>
     #region Constructors
     protected Minister() { }
 
+    /// <summary>
+    [Column("aliases")]
+    public string? Aliases { get; set; } = "";
```

Comment:
Don't make it nullable.  Set the default value in the `MinisterConfiguration` to be an empty string.